### PR TITLE
Add --skynet-api-key option

### PR DESCRIPTION
**Pull request by m-cat**
# PULL REQUEST

## Overview

- [Move from NebulousLabs to SkynetLabs and from v2.0.0 to v2.1.0](https://github.com/SkynetLabs/skynet-cli/commit/2d40817d50e9df655ed305816d2a4a42f17dfa2e)
- [Add --skynet-api-key option](https://github.com/SkynetLabs/skynet-cli/commit/1d9846ed382e8f1fba86ca309bafc321a39feecc)
- [Update docs](https://github.com/SkynetLabs/skynet-cli/commit/1bc12ad7129d95d22147604b4e4d89b42d6c5120)

## Checklist
Review and complete the checklist to ensure that the PR is complete before assigned to an approver.
 - [x] All new methods or updated methods have clear docstrings
 - [ ] Testing added or updated for new methods
 - [ ] Verify if any changes impact the WebPortal Health Checks
 - [x] Approriate documentation updated
 - [x] Changelog file created

### CODE REVIEW

**Comment by m-cat** [comment]
I've tested and confirmed this works. 👍

File: cmd/skynet/main.go [hunk 1]
```diff
@@ -58,9 +58,13 @@ var (
 	// endpointPath is the relative URL path of the endpoint.
 	endpointPath string
 
-	// apiKey is the API password to use for authentication.
+	// apiKey is the authentication password to use for a single Skynet node.
```
[the remark this answers is not in the document]
Comment (ro-tex) [reply cmd/skynet/main.go:61]: I like the clarification that this is a per-node password. I'm not sure how clear it is that it's a `skyd` password and not a portal one but this is better than before.

File: doc/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Perform actions related to Skynet
 
 ### Synopsis
 
-skynet 1.1.0
+skynet 2.0.1
```
Comment (ro-tex) [inline doc/README.md:7]: Hey, if we're releasing a new major, this is the time to make any breaking changes which we need!
Can we rename `apiKey` to `apiPassword` and remove the ambiguity once and for all?

**Review by ro-tex — APPROVED** [review]

File: doc/README.md (hunk 2 above)
Comment (kwypchlo) [reply doc/README.md:7]: I would actually rename it to skydApiPassword to be more verbose - I think anyone pointing at local skyd would be familiar enough with the prefix

File: cmd/skynet/main.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (m-cat) [reply cmd/skynet/main.go]: Made a slight change. Not sure I understood your comment tbh, let me know if it's still not clear.

[the remark this answers is not in the document]
Comment (kwypchlo) [reply cmd/skynet/main.go]: Looks good, thanks!

**Review by kwypchlo — APPROVED** [review]

**Review by MSevey — APPROVED** [review]

**Pull request merged**